TailorSort.java

```diff
@@ -43,7 +44,32 @@ private static boolean isArraySorted(Integer[] a){
         //WRITE YOUR CODE HERE
         //RETURN THE RESULT COATS ARRAY WITH ALL SORTED
         //YOU HAVE TO DO MINIMUM COMPARISONS, CLUE: USE index parameter in your sorting technique
+    	Integer key ;
+    	int swapIndex = 0;
+     	for (int i = 0; i < inputCoats.length ;i++ ){
```

Comment:
Are you using selection sort? but one part of the array is already sorted, also I have given the start index from the array is not sorted, there is no need to sort the whole array again?? or do the check again?
you can try thinking some other sorting technique just to sort this remaining unsorted array